pkg/plugins/mellanox/mellanox_plugin.go

```diff
@@ -206,3 +211,21 @@ func (p *MellanoxPlugin) nicHasExternallyManagedPFs(nicPortsMap map[string]sriov
 	}
 	return false
 }
+
+// nicNotConfiguredByOperator returns true if one of the ports(interface) of the NIC is not configured by operator
+func (p *MellanoxPlugin) nicNotConfiguredByOperator(nicPortsMap map[string]sriovnetworkv1.InterfaceExt) bool {
+	existPfStatusOnStore := false
+	for _, iface := range nicPortsMap {
+		_, exist, err := p.helpers.LoadPfsStatus(iface.PciAddress)
+		if err != nil {
+			log.Log.Error(err, "failed to load PF status from disk", "address", iface.PciAddress)
+			continue
```

Comment:
should we fail here ?

Reply:
I wasn't sure I copy the other function :P 
https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/734/files/fa266810fd4c7bc0660cbcc750ac1df761c51038#diff-8f607793a0090ecce8926d771471ecce0f2514b77695e38bbd6bb70f8996f088R200

Reply:
fair enough :)

Reply:
perhaps we shoud consider failing in both places, LMK what you think i can submit a followup PR.

LoadPfStatus will fail if file exists but it failed reading from it or its corrupt and we werent able to unmarshall

Reply:
sounds good I agree that should failed in both functions :) 

when you have time please open the PR I can review it